packages/ingest/lock.js

```diff
@@ -13,8 +13,18 @@ async function delay(t) {
 }
 
 async function countLock(bucket, pName) {
-  const list = await aws.S3.list(bucket, `${lockPrefix}/${pName}`);
-  return list.Contents.length;
+  var list = await aws.S3.list(bucket, `${lockPrefix}/${pName}`);
+  var count = list.Contents.length;
+  var item;
+  for (item in list.Contents) {
+    var date = list.Contents[item].LastModified;
+    var diff = new Date() - date;
+    if (diff > 300000) {
```

Comment:
Can we make 300000 a const and comment what unit this is in?

Reply:
Done